modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -36,15 +36,78 @@ def get_cc_appointments(user, start_date, end_date)
 
     private
 
-    def get_va_appointments_url(icn, start_date, end_date)
-      "/appointments/v1/patients/#{icn}/appointments"\
-          "?startDate=#{date_format(start_date)}&endDate=#{date_format(end_date)}&useCache=false&pageSize=0"
+    def sanitize_va_appointments(json_hash)
```

Comment:
Might be unnecessary future proofing but do we want to keep the service returning everything and have the serializer sanitize what goes to the front end?

Reply:
hmmm, thats not a bad idea tbh -- the parsing doesn't really read very well. I'm open to that, but this works for now. We need to get moving haha